auth/auth0/src/authenticate.graphql

```diff
@@ -0,0 +1,7 @@
+type AuthenticateUserPayload {
```

Comment:
To be consistent with others templates (github/facebook/google auth) this file should be named `auth0Authentication.graphql`